Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -434,5 +434,5 @@ public struct RNN<Cell: RNNCell>: Layer {
 extension RNN: Equatable where Cell: Equatable {}
 extension RNN: AdditiveArithmetic where Cell: AdditiveArithmetic {}
 
-public typealias SimpleRNN<Scalar: TensorFlowFloatingPoint> = RNN<SimpleRNNCell<Scalar>>
```

Comment:
Note: the correspondence between `SimpleRNN` and `SimpleRNNCell` seems more obvious than `SimpleRNN` and `RNNCell`. The name `SimpleRNNCell` also has precedent in [Keras](https://www.tensorflow.org/api_docs/python/tf/keras/layers/SimpleRNNCell).

Reply:
Hmm 🧐, you are right.